users/templates/partials/admin_user_update_form.html

```diff
@@ -0,0 +1,46 @@
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+<div class="row form-row">
+    <div class="form-group col-md-6 mb-0">
+        {{ form.first_name|as_crispy_field }}
+    </div>
+    <div class="form-group col-md-6 mb-0">
+        {{ form.last_name|as_crispy_field }}
+    </div>
+</div>
+<div class="row form-row">
+    <div class="form-group col-md-6 mb-0">
+        {{ form.email|as_crispy_field }}
+    </div>
+    <div class="form-group col-md-6 mb-0">
+        {{ form.user_type|as_crispy_field }}
+    </div>
+</div>
+<div class="form-group row">
+{% if admin_is_in_user_update_view %}
+    <div class="form-group col-md-4 col-md-offset-2 mb-0">
+        <input type="button" id="opener_user_account_update" class="btn btn-success" name="update"
+           value="{% trans 'Update' %}">
+    </div>
+    <div class="form-group col-md-4 mb-0">
+        <input type="button" class="btn btn-secondary discard-button" name="discard" value="{% trans 'Discard' %}"
+           onclick="location.href='{{ current_view_url }}'">
+    </div>
+{% else %}
+    <div class="form-group col-md-4 mb-0">
+        <input type="button" id="opener_user_account_update" class="btn btn-success" name="update"
+               value="{% trans 'Update' %}">
+    </div>
+    <div class="form-group col-md-4 mb-0">
+        <input type="button" class="btn btn-secondary discard-button" name="discard" value="{% trans 'Discard' %}"
+               onclick="location.href='{{ current_view_url }}'">
+    </div>
+    <div class="form-group col-md-4 mb-0">
+        <button id="change-password" type="button" class="btn btn-warning" name="password_change" onclick="location.href='{% url 'password_change' %}'">
+            <span>{% trans 'Change password' %}</span>
+        </button>
+    </div>
+{% endif %}
```

Comment:
One missing indent

Reply:
The `if` tag, right? I never know for sure how to properly format these html files with those Django tags. Do we have some sort of wiki for it?

Reply:
Done